### PR TITLE
[plugin.video.eitb] 2.0.0

### PR DESCRIPTION
**Pull request by erral**
### Description
This is an update of the already existing plugin.video.eitb plugin for Kodi Matrix
### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.eitb/.gitignore [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Byte-compiled / optimized / DLL files
```
Comment (basrieter) [inline plugin.video.eitb/.gitignore]: Please exclude from submission.

File: plugin.video.eitb/addon.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <summary lang="en">Plugin for EITB online service</summary>
+        <description lang="en">Plugin for EITB using the unofficial REST API</description>
+        <summary lang="eu">EITB Nahieranentzako plugina</summary>
+        <description lang="eu">EITB Nahieranen API ez ofiziala erabiliz egindakoa</description>
+        <summary lang="es">Plugin para EITB a la carta</summary>
+        <description lang="es">Hecho con la API no oficial de EITB a la carta</description>
+        <license>GNU General Public License, v3</license>
```
Comment (basrieter) [inline plugin.video.eitb/addon.xml]: Could you use the SPDX code for the license. See https://spdx.org/licenses/

`GPL-3.0-only`  or `GPL-3.0-or-later`

**Review by basrieter — CHANGES_REQUESTED** [review]
Could you look into the comments I gave.

**Comment by erral** [comment]
I have made the requested changes. Thanks!

**Review by basrieter — APPROVED** [review]

**Pull request merged**